ocropus-gpageseg

```diff
@@ -365,22 +365,22 @@ def process1(job):
         scale = psegutils.estimate_scale(binary)
     else:
         scale = args.scale
-    print_info("scale " + str(scale))
+    print_info("scale %f" % (scale))
     if isnan(scale) or scale>1000.0:
-        print_error("%s: bad scale (%g); skipping\n"%(fname,str(scale)))
+        print_error("%s: bad scale (%g); skipping\n" % (fname, scale))
         return
     if scale<args.minscale:
-        print_error("%s: scale (%s) less than --minscale; skipping\n"%(fname,str(scale)))
+        print_error("%s: scale (%g) less than --minscale; skipping\n" % (fname, scale))
```

Comment:
`%f` by default is more precise, `19.493589` seemed preferable to `19.4936` to me in this case. In the other instances, I just stuck with the original formatting since I couldn't test it. I'm not a big fan of exponential format.